feature-swap-impl/src/main/java/io/novafoundation/nova/feature_swap_impl/data/assetExchange/hydraDx/stableswap/model/StablePool.kt

```diff
@@ -0,0 +1,191 @@
+package io.novafoundation.nova.feature_swap_impl.data.assetExchange.hydraDx.stableswap.model
+
+import com.google.gson.Gson
+import com.google.gson.annotations.SerializedName
+import io.novafoundation.nova.common.data.network.runtime.binding.BlockNumber
+import io.novafoundation.nova.common.utils.Perbill
+import io.novafoundation.nova.common.utils.atLeastZero
+import io.novafoundation.nova.feature_swap_api.domain.model.SwapDirection
+import io.novafoundation.nova.feature_wallet_api.data.network.blockhain.assets.HydraDxAssetId
+import io.novafoundation.nova.feature_wallet_api.data.network.blockhain.types.Balance
+import io.novafoundation.nova.hydra_dx_math.stableswap.StableSwapMathBridge
+import java.math.BigInteger
+
+class StablePool(
+    val sharedAsset: StablePoolAsset,
```

Comment:
Yeah I havent implemented tradeability support yet to simplify the implementation. Shouldnt be a problem rn since currently all of the assets are fully tradable